style.css

```diff
@@ -0,0 +1,425 @@
+}
+
+.icon__line:nth-child(4) {
+  top: 26px;
+}
+
+.menu__line:first-of-type {
+  margin-bottom: 5px;
+}
+
+@media (max-width: 900px) {
+  .header__menu {
+    width: 100%;
+    display: none;
+}
+
+  .header__menu:hover {
+    background: #696969;
+}
+
+  .header__burger {
+    width: 69px;
+    height: 69px;
+}
+
+  .header__burger > span {
+    top: 29px;
+    margin-left: 21px;
+}
+
+  .header__burger > span::after {
+    content: '';
+    margin-left: 0;
+    }
+  }
+
+.new-collection {
+  position: relative;
+  display: flex;
+  min-height: 850px;
+}
+
+.new-collection__girl {
+  width: 549px;
+  height: 562px;
+  margin: 86px 54px 0 180px;
+}
+
+.new-collection__section {
+  position: relative;
+  top: 100px;
+}
+
+.new-collection__title {
+  width: 475px;
+  height: 164px;
+  margin-bottom: 36px;
+  left: 789px;
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 80px;
+  line-height: 82px;
+  color: #000;
+}
+
+.new-collection__description {
+  width: 508px;
+  margin-bottom: 36px;
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 17px;
+  line-height: 25px;
+  color: #000;
+}
+
+.new-collection__farther {
+  width: 203px;
+  height: 44px;
+  margin-right: 54px;
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 0;
+  background: none;
+  text-transform: uppercase;
+  color: #000;
+  cursor: pointer;
+  padding: 14px 41px;
+  border: 1.5px solid #000;
+  box-sizing: border-box;
+}
+
+.new-collection__farther:hover {
+  background: #d3d3d3;
+}
+
+.new-collection__other {
+  position: relative;
+  display: inline-block;
+  outline: 0;
+  border: 0;
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  background: none;
+  text-transform: uppercase;
+  color: #000;
+  cursor: pointer;
+}
+
+.new-collection__other::after {
+  content: "";
+  position: absolute;
+  background-image: url(./img/Vector%202.1.png);
+  top: 4px;
+  left: 225px;
+  width: 44px;
+  height: 12px;
+}
+
+.new-collection__other:hover {
+  transform: scale(1.2);
+}
+
+.figure {
+  position: absolute;
+  left: 50%;
+  bottom: 81px;
+}
+
+.figure__scrooll {
+  position: absolute;
+  width: 27px;
+  height: 46px;
+}
+
+.figure__strip {
+  position: absolute;
+  width: 3px;
+  height: 8.21px;
+  margin: 9px 10px 8px 11px;
+}
+
+.social-networks {
+  position: absolute;
+  bottom: 41px;
+  left: 42px;
+  transition: 1s;
+  cursor: pointer;
+}
+
+.social-networks:hover {
+  transform: scale(1.2);
+}
+
+.social__link,
+.social__link-vk {
+  margin-right: 13px;
+}
+
+@media (max-width: 565px) {
+  .adaptive {
+    display: none;
+}
+
+  .header__language {
+    display: none;
+}
+
+  .header::before {
+    display: none;
+}
+
+  .new-collection__girl {
+    display: none;
+}
+
+  .social-networks {
+    display: none;
+}
+
+  .figure {
+  display: none;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Locc/pullRequest?prid=5986341)